Water meter

Zprokkel opened the pull request:
- secondary (water) meter added
- added water topic in telegraf-dsmr.conf
- changed the m3 or m³ unit_of_measurement to use the topic

Commit:
- secondary (water) meter added
- added water topic in telegraf-dsmr.conf
- changed the m3 or m³ unit_of_measurement to use the topic

Commit:
- fixed typo

hansij66 reviewed (commented):
Hi, it looks like that in Belgium, the OBIS code for gas is slightly different from NL
You replace gas code with water vice versa.
I will not merge....but I will look at making it configurable....
Do you have a dsmr spec for Belgium?

Zprokkel commented:
Thank you for looking into this!

There isn't really much information on the P1 protocol specific for belgium let alone where it mentions water...
Sources with information I have found are:
- [extended Multi-Utility Companion Specification for the Consumer Interfaces](https://www.fluvius.be/sites/fluvius/files/2019-12/e-mucs_h_ed_1_3.pdf)
- [extended Multi‐Utility Companion Specification for the Consumer Interface P1](https://maconsosouslaloupe.be/rails/active_storage/blobs/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBbXNDIiwiZXhwIjpudWxsLCJwdXIiOiJibG9iX2lkIn19--f5c0f8166252836282d6738b3924fecc9891c9e3/e-MUCS_P1_Ed_1_3.pdf?disposition=attachment)

From what I can see it basically comes down to the following format:
Example identifier: 0, MBUS_ID, 96, 1, 1. In my case the MBUS_ID for electricity is 0 (0-0:96.1.1),  for gas is 1 (0-1:96.1.1) and water is 2 (0-2:96.1.1)

For some reason it feels to me that the installer decides what other meter (and thus mbus) to connect first. So I guess making the MBUS a variable could help the user to choose?

Zprokkel committed:
- changed code to support newer paho-mqtt version (2.10.0)
- changed code to support the newer setuptools version (82.0.1)
- added .venv to git_ignore
